oracc_rest/__init__.py

```diff
@@ -86,6 +85,26 @@ def get(self, word):
         return results
 
 
+class CombinedSuggestions(Resource):
+    def get(self, word):
+        """Call both completion and suggestion methods
+
+        This also then uses a function to combine the results into
+        a dictionary"""
+        search = ESearch()
+        completions = search.complete(word)
+        suggestions = search.suggest(word)
+        formatted = ESearch.all_suggest_compiler(completions, suggestions)
```

Comment:
This also needs to be updated now that the function is not part of `ESearch`, but I think that's all that remains to be done.

Ideally we would have a test for the combined endpoint too. This would have caught a problem like the above, for example. But that's not very easy to test with the way we have it set up, so let's leave it for now.